quantities/serialization/Injectors.cs

```diff
@@ -0,0 +1,56 @@
+using Quantities.Measures;
+
+namespace Quantities.Serialization;
+
+internal sealed class ScalarInjector : IInject
```

Comment:
There seems to be no difference between the `ScalarInjector` and the `AliasedBuilder`. Is this intended?

Reply:
I'm not quite sure I understand your question.

The `ScalarInjector` implements the interface `IInject`, whereas the `AliasedBuilder` implements `IBuilder`. They are implementations of different abstractions.

Can you perhaps clarify?

Reply:
I was mistaken. I thought the `ScalarInjector` and `AliasedBuilder` had the same implementation. I must have looked at something else.